### PR TITLE
[#221]- drop down menu added to navbar

### PR DESCRIPTION
**Pull request by FarazA22**

### CODE REVIEW

**Comment by aloverso** [comment]
Looks good, just a few small comments!

File: web/components/njwds-extended/SidebarPageLayout.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const backButton = (
   <Link href="/roadmap" passHref>
     <a
       href="/roadmap"
-      data-back-to-roadmap
+      data-back-to-roadmap="back-to-roadmap"
```
[the remark this answers is not in the document]
Comment (FarazA22) [reply web/components/njwds-extended/SidebarPageLayout.tsx:20]: Your right about that,  I've made the change.

File: web/components/navbar/NavBarLoggedInDesktop.tsx [hunk 2]
```diff
@@ -1,37 +1,110 @@
...
   const { userData } = useUserData();
   const userName = getUserNameOrEmail(userData);
 
+  const classes = useStyles();
+
+  const [open, setOpen] = useState(false);
+  const anchorRef = useRef<HTMLButtonElement | null>(null);
+
+  const handleToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
```
[the remark this answers is not in the document]
Comment (FarazA22) [reply web/components/navbar/NavBarLoggedInDesktop.tsx:27]: @aloverso The additional complexity was used to allow the browser to return focus to the button when the user presses tab to close the menu. The short answer to why this was implemented the way it was is that I used the MenuList Composition code sample as a guide. I'll look into the Popper API.

https://material-ui.com/components/menus/#menus
https://codesandbox.io/s/2f33z

[the remark this answers is not in the document]
Comment (aloverso) [reply web/components/navbar/NavBarLoggedInDesktop.tsx:27]: Sure, that seems like a fine reason to me. Just checking :)

File: web/styles/global.scss [hunk 3]
```diff
@@ -96,6 +96,15 @@ $lg: 64em; //desktop and up (1024px)
   border-color: transparent;
   cursor: pointer;
 }
+
+.text-decoration-none {
+  text-decoration: none;
```
[the remark this answers is not in the document]
Comment (FarazA22) [reply web/styles/global.scss:101]: removed from global.scss and changed class name across components

**Pull request merged**